### DISCUSSION TITLE
Introducing @vidtranslator -  A twitter bot that extracts videos, translates, and replies with a translated video

### DISCUSSION THREAD

**Discussion by altryne** [openai/whisper#232] [Show and tell]
This was inspired by @m1guel awesome subtitle script. 

I hacked together a twitter bot to help the world... better understand what's going on .

Inspired by Zelensky NATO address video, that was posted without subtitles, I ran it through whisper. 

Then I built a colab (https://github.com/altryne/whisper-me-this/) that let's you upload a video and get subtitles.

And over the weekend I decided it should be simpler! 
So now introducing the @vidtranslator bot! 
Listens to replies, extracts a video, and uses whisper to translate it and bake subtitles with ffmpeg. 

https://twitter.com/altryne/status/1576741225226698752

Would love to hear what you guys think! 

P.S. -Anyone wants to help me with hosting this on something beefier than just my machine?

**Comment by akarshghale** [comment]
Hi, Can you please share a snippet to use !auto-subtitle in .py file. I am getting an error on google colab when I use the subprocess command.
Much Appreciated!
